### DISCUSSION TITLE
Deciding on journal for manuscript

### DISCUSSION THREAD

**Discussion by wavingtowaves** [SORTEE-Github-Hackathon/manuscript#15] [General]
I'm moving our conversation about a potential journal for our manuscript from our meeting notes to this GitHub discussion thread.  

1. Nature Ecology and Evolution (They have perspective / comment articles. Would need pre-submission inquiry)
2. PLoS Biology
3. Data Science Journal
4. Ecology and Evolution
5. Trends in Ecology and Evolution (requires pre-submission inquiry)
6. Rio
7. Methods in Ecology and Evolution (would need to be non-traditional paper)
8. Ecosphere

As @emmajhudgins mentioned in our last meeting, deciding an order to the potential journals will be helpful for us to frame our paper. Also, based on comments from @LunaSare it seems like maybe we will submit a pre-submission inquiry to NEE and wait for their response.

Please indicate if there should be a different order to journals (especially in the top three spots). Thanks!

**Comment by Aariq** [comment]
I would vote for doing a pre-submission inquiry for either TREE or Nature Ecology and Evolution.

**Comment by Aguncan** [comment]
+1 to TREE

**Comment by BrandonEdwards** [comment]
I'd be wary of TREE because they have co-author maximums and fairly limited word limits. See https://www.cell.com/trends/ecology-evolution/presubmission

**Comment by wavingtowaves** [comment]
For Nature EcoEvo, here are the options for type of article we can submit: https://www.nature.com/natecolevol/content.  Based on the descriptions ours is an Article or Review Article?

**Reply by emmajhudgins** [reply]
Do you think we use enough academic literature to be considered for an article/review type? I was thinking more perspective/comment (but likely too long for a comment)
